lms/djangoapps/grades/models.py

```diff
@@ -737,6 +741,11 @@ def update_or_create_override(
             defaults=grade_defaults,
         )
 
+        override_history_entry = override.history.first()
```

Comment:
Per our conversation - I am curious to hear other opinions as well - would it make sense to add the history_user column to the persistensubsectiongradeoverride so we won't have to manually update simple history table?